### PR TITLE
added test case for testing logic of customer config reconcile for the upgrade section

### PR DESCRIPTION
**Pull request by pawelpaszki**

### CODE REVIEW

**Comment by pawelpaszki** [comment]
ping @damienomurchu @philbrookes

File: test-cases/tests/upgrade/n01-verify-customer-config-reconcile-upgrade-logic.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+estimate: 2h
+require:
+  - N01
+---
+
+# N01 - verify customer config reconcile upgrade logic
+
+## Prerequisites
+
+1. Login via `oc` as a user with **cluster-admin** role (kubeadmin):
```
Comment (matskiv) [inline test-cases/tests/upgrade/n01-verify-customer-config-reconcile-upgrade-logic.md:11]: A user from dedicate-admins group should also have the permissions we need here, but getting an oc session would have more steps, we would need to obtain a token through UI...

**Comment by pawelpaszki** [comment]
@matskiv can you please approve it?

**Comment by matskiv** [comment]
`Merging is blocked by issue 808.`
I thought that one is deleted by release pipeline.. 
I'll look into this tomorrow.

**Pull request merged**